src/api/route-services/projects.js

```diff
@@ -65,6 +66,45 @@ class ProjectsService {
     }
   }
 
+  async getProjects() {
+    const params = {
+      TableName: experimentService.experimentsTableName,
+      AttributesToGet: ['projectId'],
+    };
+
+    const dynamodb = createDynamoDbInstance();
+    const response = await dynamodb.scan(params).promise();
+    const projectIds = [];
+    response.Items.forEach((entry) => {
+      if (entry.projectId) {
```

Comment:
You coulkd apply this filter on projectId existing by using a `FilterExpression` with `attribute_exists` in the scan you have done before, then this would become just a map with `convertToJsObject`

Reply:
validation doesn't allow FilterExpression - `Can not use both expression and non-expression parameters in the same request: Non-expression parameters: {AttributesToGet} Expression parameters: {FilterExpression}`

Reply:
if you use `ProjectionExpression` instead of attributesToGet I think it should be fine.